msal-dotnet-articles/advanced/managed-identity.md

```diff
@@ -1,51 +1,79 @@
 ---
 title: Managed identity with MSAL.NET
+description: "How to use Azure managed identities in MSAL.NET applications."
 ---
 
 # Managed identity with MSAL.NET
 
 >[!NOTE]
->This is an experimental feature for 1st party use only.
+>This feature is experimental and available from [MSAL.NET](https://www.nuget.org/packages/Microsoft.Identity.Client/) version 4.51.0.
 
-## Managed Identity for Confidential Client
+A common challenge for developers is the management of secrets, credentials, certificates, and keys used to secure communication between services. [Managed identities](/azure/active-directory/managed-identities-azure-resources/overview) in Azure eliminate the need for developers to manage these credentials manually. MSAL.NET supports acquiring tokens through the managed identity capability when used with applications running inside Azure infrastructure, such as:
 
-A common challenge for developers is the management of secrets, credentials, certificates, and keys used to secure communication between services. Managed identities eliminate the need for developers to manage these credentials.
+* [Azure App Service](https://azure.microsoft.com/products/app-service/) (API version `2019-08-01` and above)
+* [Azure VMs](https://azure.microsoft.com/free/virtual-machines/)
+* [Azure Arc](/azure/azure-arc/overview)
```

Comment:
Should these be pointing to azure docs?

Reply:
@gladjohn will move that to a separate PR to address.